src/styles/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 199px;
```

Comment:
width is supposed to be 200px

Reply:
I meant 199 general width + 1 border width

Reply:
But you are using border-box property, so it includes border too. You can check it with developer tools in google chrome